src/commands/cmd_server.cc

```diff
@@ -477,7 +477,53 @@ class CommandClient : public Commander {
       }
       return Status::OK();
     }
-    return {Status::RedisInvalidCmd, "Syntax error, try CLIENT LIST|INFO|KILL ip:port|GETNAME|SETNAME"};
+
+    if ((subcommand_ == "pause")) {
+      if (args.size() != 3 && args.size() != 4) {
+        return {Status::RedisParseErr, errInvalidSyntax};
+      }
+
+      pause_timeout_ms_ = atoi(args[2].c_str());
+
+      if (args.size() == 3) {
+        pause_type_ = kPauseAll;
+      } else {
+        if (!strcasecmp(args[3].c_str(), "all")) {
```

Comment:
Can use the CommandParser to parse command arguments.

Reply:
Done!